### PR TITLE
chore(ts): Convert dropdownButton

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/dropdownButton.tsx [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-}
+type Props = React.ComponentProps<typeof Button> & {
+  isOpen?: boolean;
+  showChevron?: boolean;
+  forwardedRef?: React.Ref<typeof Button>;
+};
+
+const DropdownButton = ({
+  isOpen,
+  showChevron = false,
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/dropdownButton.tsx]: Should parameters with default values come at the end of the destucturing list?

Comment (leedongwei) [reply src/sentry/static/sentry/app/components/dropdownButton.tsx]: No preference for me. 

Might be worth discussing in the next TSC.

**Review by leedongwei — APPROVED** [review]

**Comment by leedongwei** [comment]
Approved but it'll be good to address Mark's comment before merging.

File: src/sentry/static/sentry/app/components/dropdownButton.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/dropdownButton.tsx]: I'm indifferent, I think I preserved order here. Important to note, this is a destructured object.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/dropdownButton.tsx]: Moved it to the end 🤷‍♂

**Pull request merged**